thriftserver/thriftserver/base/thriftserver-secret.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thriftserver-secret
+stringData:
+  s3_key_id: SECRET_KEY_ID
```

Comment:
It is better if you name the variables as `s3_access_key_id` and `s3_secret_access_key_id` as they better represent what we are storing in these variables

Reply:
done